File: Miscellaneous_Combination.cs
using System;
using System.Collections.Generic;

namespace Miscellaneous_CS
{
    class Program
    {
        static void Main(string[] args)
        {
            List<string> goods;
            goods = new List<string>() { "Eno", "Chidi", "Olu", "Ahmed", "Osas", "Gbeda" };
            Combination combo = new Combination();
            List<string[]> result = combo.possibleWordCombinations(goods, 3);
            // print choices and operation
            Console.Write("[ ");
            foreach (string choice in combo.words)
            {
                Console.Write(choice + "; ");
            }
            Console.WriteLine("] combination " + combo.r + ":" + Environment.NewLine);
            // print out combinations nicely
            foreach (string[] set in result)
            {
                foreach (string member in set)
                {
                    Console.Write(member + "; ");
                }
                Console.WriteLine();
            }
            Console.WriteLine(Environment.NewLine + "Number of ways is " + result.Count + ".");
        }
    }
}

File: Miscellaneous_Combination.py
#!/usr/bin/python

from Combination import Combinatorial

# Use the combination module/class
goods = ["Eno", "Chidi", "Olu", "Ahmed", "Osas", "Gbeda"]

combo = Combinatorial()
result = combo.possibleWordCombinations(goods, 3)

# print choices and operation
print("\n", combo.words, " combination ", combo.r, ":\n")

# print out combinations nicely
for group in result:
    print(group)
    
print("\n\nNumber of ways is ", len(result), ".\n")
